app/src/main/java/com/cmput301f20t21/bookfriends/ui/profile/ProfileEditDialog.java

```diff
@@ -0,0 +1,67 @@
+package com.cmput301f20t21.bookfriends.ui.profile;
+
+import android.content.Context;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.EditText;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatDialogFragment;
+import androidx.fragment.app.DialogFragment;
+
+import com.cmput301f20t21.bookfriends.R;
+
+import org.jetbrains.annotations.NotNull;
+
+public class ProfileEditDialog extends DialogFragment {
+    private EditText editEmail,editPhone;
+    private TextView cancel, confirm;
+    public ProfileEditListener listener;
```

Comment:
according to: https://www.oracle.com/java/technologies/javase/codeconventions-declarations.html
the best practice is to declare one variable per line